packages/react-components/react-storybook-addon/package.json

```diff
@@ -33,7 +33,6 @@
     "@fluentui/react-aria": "^9.13.5",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32815&commitId=57c41336c8c6c5f32bc065cfaff197c08900fdd5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|988| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2232| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>